database/factories/LessonFactory.php

```diff
@@ -21,8 +23,19 @@ class LessonFactory extends Factory
      */
     public function definition()
     {
+        $courses = Course::get();
+        $coursesId = [];
+
+        foreach ($courses as $course) {
+            array_push($coursesId, $course->id);
```

Comment:
dùng thử cái này xem em
https://stackoverflow.com/questions/34308169/eloquent-orm-laravel-5-get-array-of-ids

Reply:
em đã test thử, lần sau nếu cần fake dữ liệu kiểu kiểu như này em sẽ dùng kiểu này ạ.

Reply:
sửa luôn đi em